scripts/makeDist.sh

```diff
@@ -2,7 +2,7 @@
 VERSION="$(python ./src/version.py)"
 DEST="./dist/fpp.$VERSION.tar.gz"
 mkdir -p ./dist/
-tar -cf $DEST src/*.py fpp
+tar -czf $DEST src/*.py fpp
```

Comment:
this is `czf` in trunk and was changed recently. you probably reverted this in some kind of rebase or something -- mind dropping the change to this file?